web/app/components/comment/__control/_block/comment__control_block.scss

```diff
@@ -0,0 +1,19 @@
+.comment__control_block {
+  position: relative;
+  white-space: nowrap;
+  font-weight: 700;
+  &::after {
+    content: '▾';
+    margin-left: 2px;
+  }
+}
+
+.comment__control_block__select {
```

Comment:
It isn't a BEM. Elements of elements are prohibited.

Reply:
How it should be named?

Reply:
Usually, such names indicate that it is time to extract this part as an independent block.
I have seen the same pattern in auth-panel__sort, which seems you used as a reference.
So it is another reason to create a new component.

Reply:
Thanks, I'll take a look